resources/qml/MovieNodeTile.qml

```diff
@@ -52,6 +65,19 @@ VideoNodeTile {
             }
         }
 
+        ComboBox {
```

Comment:
![screenshot from 2018-03-11 16-00-06](https://user-images.githubusercontent.com/113075/37257875-4c91fc4e-2545-11e8-9384-eec5312f855a.png)

overflows the box for me :(